pandas/core/frame.py

```diff
@@ -6891,6 +6913,8 @@ def sort_values(
         3   48hr     40
         4   96hr     50
         >>> from natsort import index_natsorted
+        >>> index_natsorted(df["time"])
+        [0, 3, 2, 4, 1]
         >>> df.sort_values(
         ...     by="time", key=lambda x: np.argsort(index_natsorted(df["time"]))
```

Comment:
This example seems wrong. The `df["time"]` inside of the lambda should be the `x` received as the parameter. Do you mind updating it since you are improving this docstring?